test/integration/prometheus/ram_bytehours_ground_truth_test.go

```diff
@@ -35,7 +35,7 @@ func TestRAMByteHours(t *testing.T) {
 
 			// API Client
 			apiResponse, err := apiObj.GetAllocation(api.AllocationRequest{
-				Window: tc.window,
+				Window: "1d",

```

Comment:
Sorry for the confusion here, since your prom query still queries the last 24h, this should also be `"24h"`